keylime/tpm1.py

```diff
@@ -488,46 +488,27 @@ def verify_ek(self,ekcert,ekpem):
         """
         #openssl x509 -inform der -in certificate.cer -out certificate.pem
         try:
-            pubekmod = base64.b64decode(self.__get_mod_from_pem(ekpem))
```

Comment:
I don't think we can remove these lines from 491 to 504, it should be I believe lines 508 to 509 that need replacing. I will try and test this shortly.